rsc/database.properties

```diff
@@ -0,0 +1,3 @@
+jdbc.url=jdbc:mysql://localhost:3306/oh
```

Comment:
This file shouldn't be in the repo; the ones ending in `.dist` are the only one there